chameleon.py

```diff
@@ -68,20 +90,28 @@ def stop():
 def _get_cherrypy_page_handler(status_code, content_type, content):
     class Foo(object):
         @cherrypy.expose
+        @cherrypy.tools.json_out()
         def index(self, **kwargs):
             cherrypy.response.status = status_code
-            cherrypy.response.headers['Content-Type']= content_type
-            return content
+            cherrypy.response.headers['Content-Type'] = content_type
+            return json.loads(content) if 'json' in content_type else content
```

Comment:
Really, this works?  We can return a JSON-serializable dict/list structure as the content and not the JSON strnig itself?